functions/config.js

```diff
@@ -10,28 +10,28 @@
 /**
  * This is the main Backend endpoint configuration.
  * Change the values according your deployment
- * 
+ *
  * @author Mehmet Arziman - Initial contribution
  * @author Dan Cunningham - Foundations
- * 
+ *
  * host
  *    REST https host
- * 
+ *
  * port
  *    REST https port
- * 
+ *
  * userpass
  *    Optional username:password for the REST server
  *    by default oauth2 tokens will be used for authentication, uncomment this
  *    to use standard BASIC auth when talking directly to a openHAB server.
- * 
+ *
  * path
- *    Base URL path for openHAB items  
+ *    Base URL path for openHAB items
  *
  **/
 module.exports = {
 		//userpass: 'user@foo.com:Password1',
-		host: '<YOUR-CLOUD-HOST>',
+		host: 'https://openhab.thejholmes.com',
 		port: 443,
-		path: '/YOUR/REST/ENDPOINT',
+		path: '/rest',
```

Comment:
This is only valid if the Backend is openHAB or openHAB Cloud REST API. Lets keep this generic?

Reply:
Please leave it how it was before.Thx.